src/components/DataView/DataViewBulkActionsButtonsRow/DataViewBulkActionsButtonsRowTypes.ts

```diff
@@ -0,0 +1,9 @@
+import { MosaicObject } from "@root/types"
+import { DataViewBulkAction } from "../DataViewTypes"
+
+export interface DataViewBulkActionsButtonsRowProps {
+	bulkActions: DataViewBulkAction[]
+	data: MosaicObject[]
+	checked: boolean[]
+	checkedAllPages: boolean
```

Comment:
If the props  `checked` and `checkedAllPages` come from the point where the DV is instantiated I think we can reference them using its interface. `DataViewProps["checkedAllPages"];` and `DataViewProps["checked"];`